conf/application.conf

```diff
@@ -180,7 +180,7 @@ features {
 
 upscan {
   callbackUrl = "http://localhost:12600/upscan-callback"
-  maxFileSize = "5 MiB"
+  maxFileSize = "5 MB"
   maxFiles    = 5
 }
```

Comment:
Is there a reason that we have what seems to be the same value in two places?

Reply:
ones for upscan and the other is for validation - though i think i see what you mean

Reply:
Not sure I understand what you mean @amaalali. The values are different.

Reply:
Totally misread this, I thought that they were the value and human readable version